apps/builder/app/builder/features/topbar/clone.tsx

```diff
@@ -0,0 +1,30 @@
+import { useStore } from "@nanostores/react";
+import { buttonStyle, Text } from "@webstudio-is/design-system";
+import { $authToken, $authTokenPermissions } from "~/shared/nano-states";
+import { cloneProjectUrl } from "~/shared/router-utils/path-utils";
+
+export const CloneButton = () => {
+  const authTokenPermission = useStore($authTokenPermissions);
+  const authToken = useStore($authToken);
+
+  if (authToken === undefined || false === authTokenPermission.canClone) {
+    return;
+  }
+
+  return (
+    <a
```

Comment:
<Link
                      className={buttonStyle({ color: "gradient" })}
                      color="contrast"
                      underline="none"
                      href="https://webstudio.is/pricing"
                      target="_blank"
                    >
                      Upgrade
                    </Link>